### ISSUE TITLE
Fail if user tries to run multisite + data management

### ISSUE THREAD

**Issue by annawoodard** [Parsl/parsl#308]
This is not currently supported.

**Comment by kylechard** [comment]
I don't think we can currently tell if it will or won't work. It will work in multi-site if there is a shared FS (e.g., Mira/Theta) but not otherwise.

**Comment by benclifford** [comment]
Many bugs related to this have been fixed recently. #1197 is the only one outstanding at present.
